### PR TITLE
Feature/string ids

### PR DESCRIPTION
**Pull request by kostiakoval**
Add string ids support. very simple

### CODE REVIEW

File: Demo/DemoTests/Tests.m [hunk 1]
```diff
@@ -287,4 +301,33 @@ - (void)testUniquing
     [self waitForExpectationsWithTimeout:5.0f handler:nil];
 }
 
+- (void)testStringID
+{
+    XCTestExpectation *expectation = [self expectationWithDescription:@"StringID expectations"];
```
Comment (NSElvis) [inline Demo/DemoTests/Tests.m]: You can drop the async testing since DATAStack now does things synchronously

Comment (kostiakoval) [inline Demo/DemoTests/Tests.m]: tried, didn't worked out :(

Comment (kostiakoval) [inline Demo/DemoTests/Tests.m]: I'll try do sone more trick

**Comment by kostiakoval** [comment]
:tada: TADA! @NSElvis

**Comment by kostiakoval** [comment]
I've ignored Pods folder. so I think it need is it says in the documentation

**Comment by kostiakoval** [comment]
need to setup project location also

**Pull request merged**